<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>21</y>
      <width>1021</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Disparar">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>130</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disparar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AumentarAngulo">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>210</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>AumentarAngulo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DisminuirAngulo">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>280</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>DisminuirAngulo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
